spec/factories/ect_at_school_period_factory.rb

```diff
@@ -12,6 +12,16 @@
     email { Faker::Internet.email }
     working_pattern { WORKING_PATTERNS.keys.sample }
 
+    trait :future do
+      started_on { 2.weeks.from_now }
+      finished_on { nil }
+    end
+
+    trait :past do
```

Comment:
`:finished` instead of `:past`?

Reply:
I thought about this but went with `past` as it's the opposite of future`, but changing to `started` and `finished` would work too :+1: